### PR TITLE
A11Y + front-end improvements to Word Counter

### PR DESCRIPTION
**Pull request by alexsoble**
[Link to ZenHub issue.](https://app.zenhub.com/workspaces/doj-crt-intake-scrum-board-5d03bf56c1c8a35d482eca0f/issues/18f/crt-portal/155)

### For everyone

+ Improve word-count limiting functionality; enforce at 500. (Previous solution left some extra wiggle room above 500.)

### For screen readers 

+ Make reminders about word limits more persistent if user continues to type in the word counter box instead of giving a single audio warning and the then going silent.
+ Improve audio experience of word count reminders: Say, "499 words remaining" instead of "499 ... (pause) ... 499 words remaining." 

### For visual front-end 

+ Hide "0 words remaining" message when "500 word limit reached" alert is shown. 

### Screenshots (for front-end PR):

Alert state:

Screen Shot 2019-10-21 at 11 36 11 AM

### HTML changes 

+ Wrap both the word counter and the alert state in a single HTML element tied to the textarea, which helps the screenreader read out updates more accurately. 
+ Extract word counter HTML into a reusable template. 

### JS changes

+ Organize into separate functions for updating when word count not reached vs. alerting when word count reached. 

### Author

+ [ ] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.)
+ [ ] Check for [accessibility](/docs/a11y_plan.md).
+ [ ] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

### Reviewer

+ [ ] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.)
+ [ ] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

## Notes for reviewer:

See [PR instructions doc](https://github.com/usdoj/crt-portal/blob/master/docs/pull_requests.md) for full pull request review instructions.

### CODE REVIEW

**Comment by alexsoble** [comment]
Reviewed this with @Jacklynn earlier today and got her sign-off on the enhanced screen reader functionality! 🙌

**Comment by alexsoble** [comment]
@LindsayYoung, this is ready for code review. It updates both the Protected Class word counter and the Details page word counter. Please take a look when you are able!

File: crt_portal/cts_forms/templates/forms/word_counter.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<noscript>
```
Comment (LindsayYoung) [inline crt_portal/cts_forms/templates/forms/word_counter.html]: I don't think it is needed for this PR, but once we have a few of these we can put them in a folder. I like to call these "snippets" but, we can call them something better if you like.

Comment (alexsoble) [reply crt_portal/cts_forms/templates/forms/word_counter.html]: Sounds good!

**Pull request merged**